Failed to pull quay.io/metal3-io/ironic when building from scratch

Build failed when running `make` from scratch:

```
+ for IMAGE_VAR in IRONIC_IMAGE IRONIC_INSPECTOR_IMAGE
+ IMAGE=quay.io/metal3-io/ironic
+ sudo podman pull quay.io/metal3-io/ironic
Trying to pull docker://quay.io/metal3-io/ironic...time="2019-05-29T12:13:59Z" level=error msg="Error pulling image ref //quay.io/metal3-io/ironic:latest: Error determining manifest MIME type for docker://quay.io/metal3-io/ironic:latest: Error reading manifest latest in quay.io/metal3-io/ironic: unknown: Tag latest was deleted or has expired. To pull, revive via time machine" 
Failed
(0x555b17585420,0xc4207c57c0)
Error: error pulling image "quay.io/metal3-io/ironic": Invalid image name "quay.io/metal3-io/ironic", expected colon-separated transport:reference
make: *** [configure_host] Error 125
```

--- a/lib/common.sh
+++ b/lib/common.sh
@@ -48,7 +48,7 @@ export NUM_WORKERS=${NUM_WORKERS:-"1"}
 export VM_EXTRADISKS=${VM_EXTRADISKS:-"false"}
 
 # Ironic vars
-export IRONIC_IMAGE=${IRONIC_IMAGE:-"quay.io/metal3-io/ironic"}
+export IRONIC_IMAGE=${IRONIC_IMAGE:-"quay.io/metal3-io/ironic:master"}
 export IRONIC_INSPECTOR_IMAGE=${IRONIC_INSPECTOR_IMAGE:-"quay.io/metal3-io/ironic-inspector"}
 export IRONIC_DATA_DIR="$WORKING_DIR/ironic"
 

